### PR TITLE
Shopware Sample

### PR DESCRIPTION
**Pull request by tehscorpion**

### CODE REVIEW

File: shopware/Caddyfile [hunk 1]
```diff
@@ -0,0 +1,45 @@
+DOMAIN {
+    gzip {
+        level 6
```
Comment (mholt) [inline shopware/Caddyfile]: Is this really the minimal Caddyfile needed for serving shopware? Like, is there a technical reason that shopware should use gzip and have level 6 compression?

(The goal of examples is to keep them as minimal and focused as possible. Otherwise people start wondering, "why is this required? works fine for me without it")

File: shopware/Caddyfile [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        match .gif$ 1m
+        match .ico$ 1m
+        match .woff$ 1m
+        match .woff2$ 1m
+        match .ttf$ 1m
+        match .eof$ 1m
+        match .pdf$ 1s
+        match .txt$ 1s
+        match .html$ 1s
+    }
```
Comment (mholt) [inline shopware/Caddyfile]: Same question here about necessity.

Comment (tehscorpion) [inline shopware/Caddyfile]: Of course not, feel free to reduce it. Sorry :+1: 
Internal and rewrites are, but status only for some kind of additional security.

**Comment by mholt** [comment]
Take out the gzip and expires directives and I'll merge it in!

**Comment by tehscorpion** [comment]
Cool, thanks.
